examples/seq2seq/README.md

```diff
@@ -44,15 +44,18 @@ python examples/seq2seq/run_summarization.py \
     --do_train \
     --do_eval \
     --dataset_name xsum \
+    --source_prefix "summarize: " \
     --output_dir /tmp/tst-summarization \
     --per_device_train_batch_size=4 \
     --per_device_eval_batch_size=4 \
     --overwrite_output_dir \
     --predict_with_generate \
-    --max_train_samples 500 \
-    --max_val_samples 500
+    --max_train_samples 50 \
+    --max_val_samples 50
 ```
 
+Only T5 models must use an additional argument: `--source_prefix "summarize: "`.
```

Comment:
Can we be more specific here `"Only the model checkpoints [t5-small, t5-base, t5-large, t5-3b, t5-11b]"` must ... -> the "newer" t5 checkpoints should **not** be fine-tuned with the source prefix